packages/server/lib/utils/asyncWrapper.ts

```diff
@@ -13,7 +14,15 @@ export function asyncWrapper<TEndpoint extends Endpoint<any>, Locals extends Rec
 ): RequestHandler<any, TEndpoint['Reply'], any, any, any> {
     return (req, res, next) => {
         const active = tracer.scope().active();
-        active?.setTag('http.route', req.route?.path || req.originalUrl);
+        if (active) {
+            active.setTag('http.route', req.route?.path || req.originalUrl);
+            const contentLength = req.header('content-length');
+            if (contentLength) {
+                const int = parseInt(contentLength, 10);
+                active.setTag('http.request.content_length', `${(int / 1024).toPrecision(2)}kb`);
+                metrics.histogram(metrics.Types.API_REQUEST_CONTENT_LENGTH, int);
```

Comment:
not sure what the metric gives us. when would it be useful?

Reply:
it's not possible to graph the tag, but we can graph the volume of this metric so we can tie it to outage (if any)